### ISSUE TITLE
opt-level = 'z': The compilation is successful and an error is reported when running

### ISSUE THREAD

**Issue by guoyucode** [rust-lang/rust#91736]
### Code

```Rust
System: windows 10
RUST_BACKTRACE=full

[profile.release]
lto = true
opt-level = 'z'

cargo build --release

1. The compilation is correct without any errors and warnings;
2. runtime is error out println: Backtrace
3. The program is still running and does not crash
4. The logstore is no longer working: crate: log
5. If I set opt-level = 's'  , No error will be reported





```

### Meta

`rustc --version --verbose`:
```
cargo 1.57.0 (b2e52d7ca 2021-10-21)
```

### Cargo.lock

```
[Cargo.lock.txt](https://github.com/rust-lang/rust/files/7690706/Cargo.lock.txt)
[Cargo.toml.txt](https://github.com/rust-lang/rust/files/7691281/Cargo.toml.txt)
```

<details><summary><strong>Backtrace</strong></summary>
<p>

```
:
thread '311<unnamed>:' panicked at '48supplied instant is later than self
', library\std\src\time.rs:311:48
   0:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
   1:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff6e576cd24 - std::sys_common::backtrace::_print_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff6e576cd24 - std::sys_common::backtrace::_print::impl$0::fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff6e54de8ba - core::fmt::write
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\fmt\mod.rs:1163
   5:     0x7ff6e576ace8 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\io\mod.rs:1696
   6:     0x7ff6e576bdd2 - std::sys_common::backtrace::_print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff6e576bdd2 - std::sys_common::backtrace::print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff6e576bdd2 - std::panicking::default_hook::closure$1
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:210
   9:     0x7ff6e576b2e0 - std::panicking::default_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:227
  10:     0x7ff6e576ab17 - std::panicking::rust_panic_with_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:628
  11:     0x7ff6e57731d5 - std::panicking::begin_panic_handler::closure$0
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:521
  12:     0x7ff6e5773149 - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:139
  13:     0x7ff6e5773104 - std::panicking::begin_panic_handler
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:517
  14:     0x7ff6e5817730 - core::panicking::panic_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:100
  15:     0x7ff6e54e2e30 - core::panicking::panic_display<str>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:64
  16:     0x7ff6e581799b - core::option::expect_failed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\option.rs:1638
  17:     0x7ff6e5787d65 - core::option::Option::expect
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\core\src\option.rs:709
  18:     0x7ff6e5787d65 - std::time::Instant::duration_since
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:311
  19:     0x7ff6e5787d65 - std::time::impl$5::sub
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:436
  20:     0x7ff6e5787d65 - std::time::Instant::elapsed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:376
  21:     0x7ff6e55f305b - hashbrown::raw::Fallibility::alloc_err
  22:     0x7ff6e55f490b - hashbrown::raw::Fallibility::alloc_err
  23:     0x7ff6e55f0e4a - hashbrown::raw::Fallibility::alloc_err
  24:     0x7ff6e54ecaf2 - core::alloc::layout::impl$14::fmt
  25:     0x7ff6e55f071f - hashbrown::raw::Fallibility::alloc_err
  26:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  27:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  28:     0x7ff6e578987b - std::sys::windows::thread::impl$0::new::thread_start
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys\windows\thread.rs:58
  29:     0x7fffecda7034 - BaseThreadInitThunk
  30:     0x7fffeec82651 - RtlUserThreadStart
stack backtrace:
   0:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
   1:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff6e576cd24 - std::sys_common::backtrace::_print_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff6e576cd24 - std::sys_common::backtrace::_print::impl$0::fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff6e54de8ba - core::fmt::write
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\fmt\mod.rs:1163
   5:     0x7ff6e576ace8 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\io\mod.rs:1696
   6:     0x7ff6e576bdd2 - std::sys_common::backtrace::_print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff6e576bdd2 - std::sys_common::backtrace::print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff6e576bdd2 - std::panicking::default_hook::closure$1
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:210
   9:     0x7ff6e576b2e0 - std::panicking::default_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:227
  10:     0x7ff6e576ab17 - std::panicking::rust_panic_with_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:628
  11:     0x7ff6e57731d5 - std::panicking::begin_panic_handler::closure$0
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:521
  12:     0x7ff6e5773149 - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:139
  13:     0x7ff6e5773104 - std::panicking::begin_panic_handler
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:517
  14:     0x7ff6e5817730 - core::panicking::panic_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:100
  15:     0x7ff6e54e2e30 - core::panicking::panic_display<str>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:64
  16:     0x7ff6e581799b - core::option::expect_failed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\option.rs:1638
  17:     0x7ff6e5787d65 - core::option::Option::expect
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\core\src\option.rs:709
  18:     0x7ff6e5787d65 - std::time::Instant::duration_since
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:311
  19:     0x7ff6e5787d65 - std::time::impl$5::sub
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:436
  20:     0x7ff6e5787d65 - std::time::Instant::elapsed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:376
  21:     0x7ff6e55f305b - hashbrown::raw::Fallibility::alloc_err
  22:     0x7ff6e55f490b - hashbrown::raw::Fallibility::alloc_err
  23:     0x7ff6e55f0e4a - hashbrown::raw::Fallibility::alloc_err
  24:     0x7ff6e54ecaf2 - core::alloc::layout::impl$14::fmt
  25:     0x7ff6e55f071f - hashbrown::raw::Fallibility::alloc_err
  26:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  27:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  28:     0x7ff6e578987b - std::sys::windows::thread::impl$0::new::thread_start
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys\windows\thread.rs:58
  29:     0x7fffecda7034 - BaseThreadInitThunk
  30:     0x7fffeec82651 - RtlUserThreadStart
stack backtrace:
   0:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
   1:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff6e576cd24 - std::sys_common::backtrace::_print_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff6e576cd24 - std::sys_common::backtrace::_print::impl$0::fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff6e54de8ba - core::fmt::write
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\fmt\mod.rs:1163
   5:     0x7ff6e576ace8 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\io\mod.rs:1696
   6:     0x7ff6e576bdd2 - std::sys_common::backtrace::_print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff6e576bdd2 - std::sys_common::backtrace::print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff6e576bdd2 - std::panicking::default_hook::closure$1
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:210
   9:     0x7ff6e576b2e0 - std::panicking::default_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:227
  10:     0x7ff6e576ab17 - std::panicking::rust_panic_with_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:628
  11:     0x7ff6e57731d5 - std::panicking::begin_panic_handler::closure$0
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:521
  12:     0x7ff6e5773149 - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:139
  13:     0x7ff6e5773104 - std::panicking::begin_panic_handler
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:517
  14:     0x7ff6e5817730 - core::panicking::panic_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:100
  15:     0x7ff6e54e2e30 - core::panicking::panic_display<str>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:64
  16:     0x7ff6e581799b - core::option::expect_failed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\option.rs:1638
  17:     0x7ff6e5787d65 - core::option::Option::expect
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\core\src\option.rs:709
  18:     0x7ff6e5787d65 - std::time::Instant::duration_since
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:311
  19:     0x7ff6e5787d65 - std::time::impl$5::sub
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:436
  20:     0x7ff6e5787d65 - std::time::Instant::elapsed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:376
  21:     0x7ff6e55f305b - hashbrown::raw::Fallibility::alloc_err
  22:     0x7ff6e55f490b - hashbrown::raw::Fallibility::alloc_err
  23:     0x7ff6e55f0e4a - hashbrown::raw::Fallibility::alloc_err
  24:     0x7ff6e54ecaf2 - core::alloc::layout::impl$14::fmt
  25:     0x7ff6e55f071f - hashbrown::raw::Fallibility::alloc_err
  26:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  27:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  28:     0x7ff6e578987b - std::sys::windows::thread::impl$0::new::thread_start
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys\windows\thread.rs:58
  29:     0x7fffecda7034 - BaseThreadInitThunk
  30:     0x7fffeec82651 - RtlUserThreadStart
stack backtrace:
   0:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
   1:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff6e576cd24 - std::sys_common::backtrace::_print_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff6e576cd24 - std::sys_common::backtrace::_print::impl$0::fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff6e54de8ba - core::fmt::write
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\fmt\mod.rs:1163
   5:     0x7ff6e576ace8 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\io\mod.rs:1696
   6:     0x7ff6e576bdd2 - std::sys_common::backtrace::_print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff6e576bdd2 - std::sys_common::backtrace::print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff6e576bdd2 - std::panicking::default_hook::closure$1
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:210
   9:     0x7ff6e576b2e0 - std::panicking::default_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:227
  10:     0x7ff6e576ab17 - std::panicking::rust_panic_with_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:628
  11:     0x7ff6e57731d5 - std::panicking::begin_panic_handler::closure$0
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:521
  12:     0x7ff6e5773149 - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:139
  13:     0x7ff6e5773104 - std::panicking::begin_panic_handler
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:517
  14:     0x7ff6e5817730 - core::panicking::panic_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:100
  15:     0x7ff6e54e2e30 - core::panicking::panic_display<str>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:64
  16:     0x7ff6e581799b - core::option::expect_failed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\option.rs:1638
  17:     0x7ff6e5787d65 - core::option::Option::expect
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\core\src\option.rs:709
  18:     0x7ff6e5787d65 - std::time::Instant::duration_since
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:311
  19:     0x7ff6e5787d65 - std::time::impl$5::sub
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:436
  20:     0x7ff6e5787d65 - std::time::Instant::elapsed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:376
  21:     0x7ff6e55f305b - hashbrown::raw::Fallibility::alloc_err
  22:     0x7ff6e55f490b - hashbrown::raw::Fallibility::alloc_err
  23:     0x7ff6e55f0e4a - hashbrown::raw::Fallibility::alloc_err
  24:     0x7ff6e54ecaf2 - core::alloc::layout::impl$14::fmt
  25:     0x7ff6e55f071f - hashbrown::raw::Fallibility::alloc_err
  26:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  27:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  28:     0x7ff6e578987b - std::sys::windows::thread::impl$0::new::thread_start
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys\windows\thread.rs:58
  29:     0x7fffecda7034 - BaseThreadInitThunk
  30:     0x7fffeec82651 - RtlUserThreadStart
stack backtrace:
   0:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
   1:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff6e576cd24 - std::sys_common::backtrace::_print_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff6e576cd24 - std::sys_common::backtrace::_print::impl$0::fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff6e54de8ba - core::fmt::write
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\fmt\mod.rs:1163
   5:     0x7ff6e576ace8 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\io\mod.rs:1696
   6:     0x7ff6e576bdd2 - std::sys_common::backtrace::_print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff6e576bdd2 - std::sys_common::backtrace::print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff6e576bdd2 - std::panicking::default_hook::closure$1
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:210
   9:     0x7ff6e576b2e0 - std::panicking::default_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:227
  10:     0x7ff6e576ab17 - std::panicking::rust_panic_with_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:628
  11:     0x7ff6e57731d5 - std::panicking::begin_panic_handler::closure$0
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:521
  12:     0x7ff6e5773149 - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:139
  13:     0x7ff6e5773104 - std::panicking::begin_panic_handler
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:517
  14:     0x7ff6e5817730 - core::panicking::panic_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:100
  15:     0x7ff6e54e2e30 - core::panicking::panic_display<str>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:64
  16:     0x7ff6e581799b - core::option::expect_failed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\option.rs:1638
  17:     0x7ff6e5787d65 - core::option::Option::expect
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\core\src\option.rs:709
  18:     0x7ff6e5787d65 - std::time::Instant::duration_since
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:311
  19:     0x7ff6e5787d65 - std::time::impl$5::sub
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:436
  20:     0x7ff6e5787d65 - std::time::Instant::elapsed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:376
  21:     0x7ff6e55f305b - hashbrown::raw::Fallibility::alloc_err
  22:     0x7ff6e55f490b - hashbrown::raw::Fallibility::alloc_err
  23:     0x7ff6e55f0e4a - hashbrown::raw::Fallibility::alloc_err
  24:     0x7ff6e54ecaf2 - core::alloc::layout::impl$14::fmt
  25:     0x7ff6e55f071f - hashbrown::raw::Fallibility::alloc_err
  26:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  27:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  28:     0x7ff6e578987b - std::sys::windows::thread::impl$0::new::thread_start
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys\windows\thread.rs:58
  29:     0x7fffecda7034 - BaseThreadInitThunk
  30:     0x7fffeec82651 - RtlUserThreadStart
thread '<unnamed>' panicked at 'supplied instant is later than self', library\std\src\time.rs:311:48
stack backtrace:
   0:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::dbghelp::trace
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
   1:     0x7ff6e576cd24 - std::backtrace_rs::backtrace::trace_unsynchronized
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
   2:     0x7ff6e576cd24 - std::sys_common::backtrace::_print_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:67
   3:     0x7ff6e576cd24 - std::sys_common::backtrace::_print::impl$0::fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:46
   4:     0x7ff6e54de8ba - core::fmt::write
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\fmt\mod.rs:1163
   5:     0x7ff6e576ace8 - std::io::Write::write_fmt<std::sys::windows::stdio::Stderr>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\io\mod.rs:1696
   6:     0x7ff6e576bdd2 - std::sys_common::backtrace::_print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:49
   7:     0x7ff6e576bdd2 - std::sys_common::backtrace::print
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:36
   8:     0x7ff6e576bdd2 - std::panicking::default_hook::closure$1
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:210
   9:     0x7ff6e576b2e0 - std::panicking::default_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:227
  10:     0x7ff6e576ab17 - std::panicking::rust_panic_with_hook
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:628
  11:     0x7ff6e57731d5 - std::panicking::begin_panic_handler::closure$0
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:521
  12:     0x7ff6e5773149 - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys_common\backtrace.rs:139
  13:     0x7ff6e5773104 - std::panicking::begin_panic_handler
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\panicking.rs:517
  14:     0x7ff6e5817730 - core::panicking::panic_fmt
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:100
  15:     0x7ff6e54e2e30 - core::panicking::panic_display<str>
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\panicking.rs:64
  16:     0x7ff6e581799b - core::option::expect_failed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\core\src\option.rs:1638
  17:     0x7ff6e5787d65 - core::option::Option::expect
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\core\src\option.rs:709
  18:     0x7ff6e5787d65 - std::time::Instant::duration_since
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:311
  19:     0x7ff6e5787d65 - std::time::impl$5::sub
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:436
  20:     0x7ff6e5787d65 - std::time::Instant::elapsed
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\time.rs:376
  21:     0x7ff6e55f305b - hashbrown::raw::Fallibility::alloc_err
  22:     0x7ff6e55f490b - hashbrown::raw::Fallibility::alloc_err
  23:     0x7ff6e55f0e4a - hashbrown::raw::Fallibility::alloc_err
  24:     0x7ff6e54ecaf2 - core::alloc::layout::impl$14::fmt
  25:     0x7ff6e55f071f - hashbrown::raw::Fallibility::alloc_err
  26:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  27:     0x7ff6e578987b - alloc::boxed::impl$44::call_once
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\library\alloc\src\boxed.rs:1691
  28:     0x7ff6e578987b - std::sys::windows::thread::impl$0::new::thread_start
                               at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c\/library\std\src\sys\windows\thread.rs:58
  29:     0x7fffecda7034 - BaseThreadInitThunk
  30:     0x7fffeec82651 - RtlUserThreadStart
```

</p>
</details>
